app/components/common/Navbar.js

```diff
@@ -0,0 +1,24 @@
+import React, {Component} from 'react'
+import ConnectionForm from '../connectionForm';
+import Indicator from '../indicator';
+
+export default class Navbar extends Component {
+  render() {
+    return(
+      <nav className="navbar navbar-default navbar-fixed-top">
+        <div className="container">
+          <div className="navbar-header">
+            <a className="navbar-brand" href="#">Rbkit Client</a>
+          </div>
+          <div id="navbar">
+            <ul className="nav navbar-nav navbar-right">
```

Comment:
@kgrz Can move but I think this markup fits for this component since we extracted the `Navbar` to a separate component or we can remove this whole component and put the markup in layout itself.